next.config.js

```diff
@@ -33,6 +27,12 @@ module.exports = withMDX({
     // config.resolve.alias["react/jsx-dev-runtime"] = require.resolve('react/jsx-dev-runtime');
     // config.resolve.alias["react/jsx-runtime"] = require.resolve('react/jsx-runtime');
 
-    return config
+    // Add the loader configuration for the problematic file type
+    config.module.rules.push({
```

Comment:
instead of making changes in next config and loading yaml in frontend code, better put a script in https://github.com/asyncapi/website/tree/master/scripts that will run in build time (just add script to package.json)

Reply:
I would suggest kindly use yaml to json converter and append the json as the part of `posts.json`. This json will then be used to render the tsc component. Website should not render yaml in its direct form.

Reply:
Also, if we are creating a script for rendering it inside website, I don't think we need to make extra script inside package.json. The script should be part of existing script of dev and start only. You can append your script by adding the function inside `scripts/index.js`.